Bottom: 471aba4b7cfa2c7039a9d7ae6ddc7b5b0280d18f
Top:    e84588ff9b376b7b0be62cf7d4be0fde827308f4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-03 01:33:35 +0000

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index d35b913..afd0b86 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -173,7 +173,6 @@ ghid_set_layer (const char *name, int group, int empty)
 
   hidgl_set_depth (compute_depth (group));
 
-//  glFlush ();
   glEnable (GL_STENCIL_TEST);                   // Enable Stencil test
   glStencilOp (GL_KEEP, GL_KEEP, GL_REPLACE);   // Stencil pass => replace stencil value (with 1)
   hidgl_return_stencil_bit (stencil_bit);       // Relinquish any bitplane we previously used
@@ -187,7 +186,6 @@ ghid_set_layer (const char *name, int group, int empty)
     glStencilFunc (GL_ALWAYS, 0, 0);  // Always pass stencil test
   }
 
-
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
       priv->trans_lines = true;
@@ -1035,8 +1033,6 @@ ghid_start_drawing (GHidPort *port)
   if (!gdk_gl_drawable_gl_begin (pGlDrawable, pGlContext))
     return FALSE;
 
-//  hidgl_load_frag_shader ();
-
   return TRUE;
 }
